lua/lazy-lsp/servers.lua

```diff
@@ -152,7 +152,7 @@ return {
   marksman = "marksman",
   matlab_ls = "matlab-language-server",
   mdx_analyzer = "",
-  metals = "metals",
+  metals = { "metals", "jdk17" },
```

Comment:
Would it make sense to just use `jdk` so we don't have to update versions later?

Reply:
Good question. In the Scala ecosystem, the tools don't always just work with the latest jdk version. That's why I decided to use an older LTS version here. But I'm also not perfectly happy with that choice. Any ideas?

Reply:
fair enough, we can always bump it in the future if it starts causing issues